server/services/recalculate-rejection-reasons-occurrences.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+var mongo    = require('../mongo-db')
+  , Set      = require('es6-set')
+  , deferred = require('deferred');
+
+module.exports = function () {
```

Comment:
@mtuchowski no it's not. It is supposed to update `rejectionReasonsConcat` on every server start in order to facilitate grouping query search.

Check out the adjusts...